localflavor/kw/kw_governorates.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.utils.translation import ugettext_lazy as _
+
+# List of governorates as defined by ISO_3166-2
+# For reference; see https://en.wikipedia.org/wiki/ISO_3166-2:KW
+# Note that these are governorates and not subdivisons
```

Comment:
There's a typo in the comment: 'subdivisons' -> 'subdivisions'.

Does this comment just meant that the name for these boundaries is 'governorate', not 'subdivision'? Or are governorates something different than subdivisions?